Utilities/bootstrap

```diff
@@ -972,86 +972,88 @@ def main():
         if result != 0:
             error("tests failed with exit status %d" % (result,))
 
-    # If installing, put the build products in the appropriate location.
+    # If installing, put the build products in the appropriate locations.
     if "install" in build_actions:
-        bin_install_path = os.path.join(g_project_root, args.install_prefix,
-                                        "bin")
-        lib_install_path = os.path.join(g_project_root, args.install_prefix,
-                                        "lib", "swift", "pm")
-        libexec_install_path = os.path.join(g_project_root, args.install_prefix,
-                                        "libexec", "swift", "pm")
-        mkdir_p(bin_install_path)
-        mkdir_p(lib_install_path)
-        mkdir_p(libexec_install_path)
-
-        # Install a binary file at the install path.
-        def installBinary(binary_path, install_path, add_rpath=None):
-            cmd = ["install", binary_path, install_path]
-            note("installing %s: %s" % (
-                os.path.basename(binary_path), ' '.join(cmd)))
-            result = subprocess.call(cmd)
-            if result != 0:
-                error("install failed with exit status %d" % (result,))
-                
-            # Remove the hard-coded stdlib RPATHs that `swiftc` bakes in on
-            # Darwin.
-            #
-            # FIXME: We need a way to control this, instead of having to rip it
-            # out after the fact. https://bugs.swift.org/browse/SR-1967
-            if platform.system() == 'Darwin':
-                installed_path = os.path.join(
-                    install_path, os.path.basename(binary_path))
-                stdlib_path = os.path.realpath(
-                    os.path.join(os.path.dirname(args.swiftc_path), "..",
-                                 "lib", "swift", "macosx"))
-                cmd = ["install_name_tool", "-delete_rpath",
-                       stdlib_path, installed_path]
-                note("removing stray RPATH from %s: %s" % (
-                        installed_path, ' '.join(cmd)))
+        # Install in each prefix.
+        for install_prefix in args.install_prefixes:
```

Comment:
FYI, for diff readability I probably would have factored this into a separate function in a single NIFC (no intended functionality change) commit, then made the functionality change.